README.md

```diff
@@ -144,7 +144,7 @@ export ANDROID_TOOLCHAIN_NAME=arm-linux-androideabi-clang
 # pull and build ros2 for android
 mkdir -p ${ROS2_ANDROID_WORKSPACE}/src
 cd ${ROS2_ANDROID_WORKSPACE}
-wget https://raw.githubusercontent.com/esteve/ros2_java/master/ros2_java_android.repos
+wget https://raw.githubusercontent.com/esteve/ros2_java/dashing/ros2_java_android.repos
```

Comment:
@nielstiben and also here https://raw.githubusercontent.com/ros2-java/ros2_java/dashing/ros2_java_android.repos? Thanks

Reply:
@esteve also updated.